--- a/analyzer/Makefile
+++ b/analyzer/Makefile
@@ -29,11 +29,10 @@
 $(ANALYZER): analyzer.c pgn.c analyzer.h pgn.h ../common/common.c ../common/common.h Makefile
 	$(CC) $(CPPFLAGS) $(CFLAGS) $(LDFLAGS) -o $(ANALYZER) -I../common pgn.c analyzer.c ../common/common.c $(LDLIBS$(LDLIBS-$(@)))
 
+# NOTE: What is going on here any why? -pjf
 json: $(ANALYZER) pgns2json.xslt
 	$(ANALYZER) -explain-xml >$(XMLFILE) && xsltproc pgns2json.xslt $(XMLFILE) >$(JSONFILE)
 
 clean:
 	-rm -f $(TARGETS) *.elf *.gdb
-	-rm -f $(XMLFILE)
-	-rm -f $(JSONFILE)
 
